lib/ansiblelint/_internal/config.py

```diff
@@ -0,0 +1,9 @@
+"""Loads config options."""
+import sys
+
+from ansiblelint import cli
+
+if any(i in ['pytest', 'sphinx'] for i in sys.modules):
+    options = cli.get_config([])
+else:
+    options = cli.get_config(sys.argv[1:])
```

Comment:
This is a bad approach because it heavily relies on the global state of `sys.argv` and `sys.modules`. I don't see how this code can be treated as predictable.